boatd-opencpn

```diff
@@ -79,6 +80,10 @@ def send_nmea_message(sock, message, dest):
 
 
 def main(ip, port, delay):
+    parser = argparse.ArgumentParser()
+    parser.add_argument("endpoint", help="boatd API endpoint")
+    endpoint = split_endpoint(parser.parse_args().endpoint)
```

Comment:
Mostly looks good! You'll also want to pass this information to the boatd connection, something like

    boatd = boatdclient.Boatd(host, port)

and pass it to the `boat` initializer:

    boat = boatdclient.Boat(boatd=boatd, auto_update=False)

is `split_endpoint` supposed to be in this commit?